pkg/client/openstack/admin/client.go

```diff
@@ -21,13 +21,14 @@ import (
 	"github.com/sapcc/kubernikus/pkg/client/openstack/roles"
 )
 
-var serviceUserRoles = []string{"network_admin", "member"}
+var serviceUserRoles = []string{"network_admin", "member", "swiftoperator"}
```

Comment:
We should not assign the `swiftoperator` role to the kubernikus service user. This allows read/write access to all swift data in the project. Instead we need to create the audit container and give the user access to this container only, using the swift acls.
Are are already doing this for the etcd backup using the admin client here: https://github.com/sapcc/kubernikus/blob/3d768210318ba91d10b3f1746f29520c39f62492/pkg/client/openstack/admin/client.go#L346

I'm not sure if we are reconciling the swift container but if we don't I would suggest that we do that for both containers in ground control. (E.g. in the running phase if the etcd backup is enabled we make sure the storage container is (re)created, if the audit logging is enabled we (re)create the audit container.